### PR TITLE
Added jupyter notebook for Geospatial Blog

### PR DESCRIPTION
**Pull request by philippkahr**
Hi,

this should add the geospatial Jupyter Notebook for the accompanying blog post.

### CODE REVIEW

**Comment by joemcelroy** [comment]
Hi there,

Theres an issue rendering the notebook (https://github.com/elastic/elasticsearch-labs/blob/c9d54687840cac94f94dffa5ac4e4e76a9c7f2ce/notebooks/search/09-geospatial-search.ipynb) - looks like you have images in the notebook that doesn't exist.

Also as this is for content for blog, could you add it in the supporting-blog-content folder instead please?

File: supporting-blog-content/geospatial-llm/09-geospatial-search.ipynb [hunk 1]
```diff
@@ -0,0 +1,2072 @@
...
+    "Now let's finally get to the AI part. All of this was the setup and understanding what makes geo spatial searches tick and how they work. There is still a lot more to discover. Let's hookup it up to our OpenAI instance. In here we use the Azure OpenAI resource."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We are using the Azure OpenAI Ressource.\n",
```
Comment (carlyrichmond) [inline supporting-blog-content/geospatial-llm/09-geospatial-search.ipynb:1465]:
```suggestion
    "# We are using the Azure OpenAI Resource.\n",
```

**Review by carlyrichmond — APPROVED** [review]
I found one minor typo but aside from that it's looking good.

**Comment by carlyrichmond** [comment]
@philippkahr I think you've got a formatting issue still on the notebook as it's still not building:

> <unknown>:11: SyntaxWarning: invalid escape sequence '\.'
<unknown>:11: SyntaxWarning: invalid escape sequence '\.'
<unknown>:11: SyntaxWarning: invalid escape sequence '\.'
error: cannot format supporting-blog-content/geospatial-llm/09-geospatial-search.ipynb: Cannot parse: 37:46:     print(f"Distance between AirBnB and {hit["_source"]["name"]}: {hit["sort"][0]} km")

**Pull request merged**